src/Tasks.js

```diff
@@ -0,0 +1,57 @@
+let tasksArray = JSON.parse(localStorage.getItem('task')) || [];
+const input = document.querySelector('.input');
+const todoList = document.querySelector('.todo-list');
+
+export default class Task {
+  constructor(description) {
+    this.index = tasksArray.length + 1;
+    this.description = description;
+    this.completed = false;
+  }
+}
+
+const populateList = (tasksArray) => {
+  todoList.innerHTML = '';
+  for (let i = 0; i < tasksArray.length; i += 1) {
+    const li = document.createElement('li');
+    li.id = `${tasksArray[i].index}`;
+    li.innerHTML = `<input type="checkbox"><div class="task-text" contenteditable="true" data_id='${i}'>${tasksArray[i].description}</div>
+        <i class="fas fa-trash-alt delete" data_id='${i}'></i>`;
```

Comment:
Deleting a task should update all remaining items' indexes, so they represent the current list order and are unique.
Kindly see the screenshot below which shows that indexes are not unique after deleting an index.
![indexes](https://user-images.githubusercontent.com/35504149/165807609-37c40d2a-6844-4770-b9ec-7b2f8d4d21d1.png)